lib/gettext/plural.ex

```diff
@@ -681,4 +728,52 @@ defmodule Gettext.Plural do
       _other -> raise UnknownLocaleError, locale
     end
   end
+
+  @doc false
+  def pluralization_opts(locale, messages_struct, plural_mod) do
+    {:module, ^plural_mod} = Code.ensure_loaded(plural_mod)
```

Comment:
Should this give a better error message if the `plural_mod` is not loaded, explaining why it needs to be loaded?

Reply:
@whatyouhide I needed to put it into place since it is loaded automatically if a function is called but not to check with `function_exported?`.

We can yield an error though if it cant be loaded.

Something like this?

```suggestion
    case Code.ensure_loaded(plural_mod) do
      {:module, ^plural_mod} -> :ok
      _other -> raise "The module #{plural_mode} can't be found." # Better message?
    end
```

Reply:
@josevalim I initially used that one. Unfortunately, this lib supports 1.11 and `ensure_loaded!` was introduced in 1.12.

Reply:
@maennchen define `ensure_loaded!/1` in this module conditionally, something like this

```elixir
# TODO: remove when we depend on Elixir 1.12+
if function_exported?(Code, :ensure_loaded!, 1) do
  defp ensure_loaded!(mod), do: Code.ensure_loaded!(mod)
else
  defp ensure_loaded!(mod) do
    case Code.ensure_loaded(mod) do
      {:module, ^mod} -> :ok
      _other -> raise "<good error message>"
    end
  end
end
```